conan/llvm/conanfile.py

```diff
@@ -91,16 +90,15 @@ class LLVMConan(ConanFile):
 
     def source(self):
         git_cache = os.environ.get("CONAN_LLVM_GIT_CACHE")
-        cache_hit = lambda: os.path.exists(f"{git_cache}/.git")
+        cache_hit = os.path.exists(f"{git_cache}/.git")
```

Comment:
If you pulled this out above (L22-25), then why keep the redefinition here?

Reply:
Was a leftover attempt at a flake fix. I've removed.